lib/cocoapods/generator/copy_xcframework_script.rb

```diff
@@ -140,7 +140,7 @@ def script
   select_slice "${paths[@]}"
   local target_path="$SELECT_SLICE_RETVAL"
   if [[ -z "$target_path" ]]; then
-    echo "warning: [CP] Unable to find matching .xcframework slice in '${paths[@]}' for the current build architectures ($ARCHS)."
+    echo "error: [CP] Unable to find matching .xcframework slice in '${paths[@]}' for the current build architectures ($ARCHS)."
```

Comment:
actually I think I remember why I made this a warning - there can be cases where a framework is only used in certain build variants (ex. a framework that is only used on either Simulator or device, but not both) and I wasn't confident that we should block the build 100% of the time

Reply:
That is a good idea for no-block, but if the framework do not contain the target arch,at last it will sumit a `undefine symbol` error.

Reply:
Only if you are using that symbol in that architecture. What I am describing is something where you might have a framework that you only use on device, and then your code looks like this:

```
#if CAN_USE_DEVICE_FRAMEWORK_XYZ
import XYZ
...
#endif
```

Reply:
it will cause cannot found header file error
for example
i have a xcframework like this
- a.xcframework
 - arm64_armv7.framework
 - x86_64-simulartor
when run simulartor destination
it wil no copy any framework to PODS_XCFRAMEWORKS_BUILD_DIR and just echo a warning "unable to ...."
when compile a file include "a/xx.h" will cause cannot find `a/xx.h`,becase PODS_XCFRAMEWORKS_BUILD_DIR has no `a/xx.h`